src/platform/site-wide/index.js

```diff
@@ -18,13 +18,21 @@ import startVAFooter from './va-footer';
 
 import brandConsolidation from '../brand-consolidation';
 
+import isMetricsEnabled from '../frontend-metrics/feature-flag';
+import withinMetricsSample from '../frontend-metrics/sample-rate';
+import startMetrics from '../frontend-metrics/metrics';
+
 /**
  * Start up the site-wide components that live on every page, like
  * the login widget, the header menus, and the feedback widget.
  *
  * @param {Store} commonStore The Redux store being used by this application
  */
 export default function startSitewideComponents(commonStore) {
+  if (isMetricsEnabled() && withinMetricsSample()) {
```

Comment:
I think I would favor putting a single function call in this file and doing this logic in code that lives with the metrics files. It's easier if we keep this file as simple as we can (I already dislike how much conditional logic is in here).